.github/workflows/test_ryzenai_quantization_timm.yaml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Ryzen Timm Quantization - Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+    runs-on: [self-hosted, single-npu, amd-npu, ryzenai]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Create and start a virtual environment
+      run: |
+        conda create --prefix venv --clone C:\tools\Anaconda3\envs\ryzenai-1.0.1-ci
+        conda init
+        conda activate .\venv
+    - name: Install dependencies
+      run: |
+        conda activate .\venv
+        python -m pip install --upgrade pip
+        pip install .[tests]
+        pip install git+https://github.com/huggingface/optimum.git
+    - name: Test with Pytest
+      run: |
+        conda activate .\venv
+        huggingface-cli login --token ${{ secrets.HF_READ_TOKEN }}
+        $env:XLNX_VART_FIRMWARE="C:\RyzenAiSw\ryzen-ai-sw-1.0.1\voe-4.0-win_amd64\1x4.xclbin"
+        pytest -s -v -m "not run_slow" tests/ryzenai/test_quantization.py 
```

Comment:
Instead of `not run_slow`, can we keep it like transformers which uses `RUN_SLOW=1` to run tests?

And the `@slow` decorator checks the `RUN_SLOW` env variable

Reply:
Yes, its using the same `@slow` decorator. But this `run_slow` marker was also added, updated this marker to `quant_test`